src/workflow/client/DaprWorkflowClient.ts

```diff
@@ -0,0 +1,201 @@
+    workflowInstanceId: string,
+    getInputsAndOutputs: boolean,
+  ): Promise<WorkflowState | undefined> {
+    const state = await this._innerClient.getOrchestrationState(workflowInstanceId, getInputsAndOutputs);
+    if (state !== undefined) {
+      return new WorkflowState(state);
+    }
+  }
+
+  /**
+   * Waits for a workflow to start running and returns a {@link WorkflowState} object
+   * containing metadata about the started instance, and optionally, its input, output,
+   * and custom status payloads.
+   *
+   * A "started" workflow instance refers to any instance not in the Pending state.
+   *
+   * If a workflow instance is already running when this method is called, it returns immediately.
+   *
+   * @param {string} workflowInstanceId - The unique identifier of the workflow instance to wait for.
+   * @param {boolean} fetchPayloads - Indicates whether to fetch the workflow instance's
+   *                                  inputs, outputs (true) or omit them (false).
+   * @param {number} timeoutInSeconds - The amount of time, in seconds, to wait for the workflow instance to start.
+   * @returns {Promise<WorkflowState | undefined>} A Promise that resolves to the workflow instance metadata
+   *                                               or undefined if no such instance is found.
+   */
+  public async waitForWorkflowStart(
+    workflowInstanceId: string,
+    fetchPayloads = true,
+    timeoutInSeconds = 60,
+  ): Promise<WorkflowState | undefined> {
+    const state = await this._innerClient.waitForOrchestrationStart(
+      workflowInstanceId,
+      fetchPayloads,
+      timeoutInSeconds,
+    );
+    if (state !== undefined) {
+      return new WorkflowState(state);
+    }
+  }
+
+  /**
+   * Waits for a workflow to complete running and returns a {@link WorkflowState} object
+   * containing metadata about the completed instance, and optionally, its input, output,
+   * and custom status payloads.
+   *
+   * A "completed" workflow instance refers to any instance in one of the terminal states.
+   * For example, the Completed, Failed, or Terminated states.
+   *
+   * If a workflow instance is already running when this method is called, it returns immediately.
+   *
+   * @param {string} workflowInstanceId - The unique identifier of the workflow instance to wait for.
+   * @param {boolean} fetchPayloads - Indicates whether to fetch the workflow instance's
+   *                                  inputs, outputs (true) or omit them (false).
+   * @param {number} timeoutInSeconds - The amount of time, in seconds, to wait for the workflow instance to start.
+   * @returns {Promise<WorkflowState | undefined>} A Promise that resolves to the workflow instance metadata
+   *                                               or undefined if no such instance is found.
+   */
+  public async waitForWorkflowCompletion(
+    workflowInstanceId: string,
+    fetchPayloads = true,
+    timeoutInSeconds = 60,
```

Comment:
Is it required here? https://github.com/microsoft/durabletask-js/blob/df277ac25489e67e3423a7f9d2475ad1d2c6a184/src/client/client.ts#L161 already defaults to 60.

Reply:
I think it's good to keep it here as it will be easier for users to find the default timeout as most will directly use js-sdk not durabletask-js.